apps/Gruntfile.js

```diff
@@ -568,6 +564,17 @@ describe('entry tests', () => {
     }
   };
 
+  config.clean = {
+    all: ['build'],
```

Comment:
For my understanding, are there docs I can look at to understand how this `all` attribute is consumed? And how you figured out the right `unitTest` config below?

Reply:
This `clean` operation is managed by the `grunt-contrib-clean` package; I just consulted their documentation at https://github.com/gruntjs/grunt-contrib-clean. Note that this PR isn't actually adding this config block or the `all` configuration, just moving them from further up in the file (specifically so this block comes after the definition of the `webpackOutputBasePath` variable which it references)

What I specifically add here is just the standard "per-target options" format from https://github.com/gruntjs/grunt-contrib-clean#options-1 with the `force` flag described at https://github.com/gruntjs/grunt-contrib-clean#options to allow this to work outside of our current working directory (in this case, in `/tmp/`)

Reply:
Yep I'm with you re: not a new block. Thanks for the links! Was wondering about what that `all` configuration represents and how it might interact with your new config, but seems like they'd be pretty independent operations.

Reply:
Yep, I think so! I don't think there's anything particularly special about the `all` configuration, it's just the name we chose to represent our default category. I suppose we could consider renaming it to something like "default"; doesn't really make sense to have the category be called "all" now that there's something that's not in it

Reply:
Ok I just did a little playing around to solidify my understanding, I see now that if you run `npm run clean` (ie, `grunt clean`) we run both tasks, which seems desirable. Was wondering how we specifically ran the `all` configuration, didn't realize that the regular `grunt clean` will run all tasks configured (mostly writing this out for anyone else looking at this)

```
Bens-Laptop:apps benjaminbrooks$ npm run clean

> blockly-mooc@0.0.161 clean /Users/benjaminbrooks/code-dot-org/apps
> grunt clean

Running "clean:all" (clean) task
>> 1 path cleaned.

Running "clean:unitTest" (clean) task
>> 1 path cleaned.

Done.
```